src/failprint/capture.py

```diff
@@ -113,8 +113,15 @@ def __init__(self, capture: Capture = Capture.BOTH, stdin: str | None = None) ->
         self._saved_stdout_fd: int = -1
         self._saved_stderr_fd: int = -1
         self._output: str | None = None
+        self._old_env: dict[str, str | None] = {}
 
     def __enter__(self) -> CaptureManager:  # noqa: PYI034 (false-positive)
+        # Annoying operating system being annoying again.
+        # Bash ports and WSL2 work fine, this environment variable
+        # just helps a bit with cmd and PowerShell.
+        self._old_env["PYTHONLEGACYWINDOWSSTDIO"] = os.getenv("PYTHONLEGACYWINDOWSSTDIO")
+        os.environ["PYTHONLEGACYWINDOWSSTDIO"] = "1"
```

Comment:
Kay, testing without the fix now.

```suggestion
        #self._old_env["PYTHONLEGACYWINDOWSSTDIO"] = os.getenv("PYTHONLEGACYWINDOWSSTDIO")
        #os.environ["PYTHONLEGACYWINDOWSSTDIO"] = "1"
```